main.py

```diff
@@ -173,7 +201,25 @@ def main():
 		DISPLAY.blit(ballVelLabel, (10, 30))
 		mouseVelLabel = font.render("Mouse velocity: {:.2f}ms-1".format(mouseBall.velocity.mag), 1, BLACK)
 		DISPLAY.blit(mouseVelLabel, (10, 50))
-
+		gravityVelLabel = font.render("Gravity: {}ms-2".format(GRAVITY), 1, BLACK)
+		DISPLAY.blit(gravityVelLabel, (10, 70))
+		diameterVelLabel = font.render("Ball diameter: {:.2f}m".format(ball.radius*2), 1, BLACK)
+		DISPLAY.blit(diameterVelLabel, (10, 90))
+		stringVelLabel = font.render("String length: {:.2f}m".format(string.length), 1, BLACK)
+		DISPLAY.blit(stringVelLabel, (10, 110))
+
+
+		#control prompts
+		stringConnect = font.render("Connect string:           SPACE", 1, BLACK)
+		DISPLAY.blit(stringConnect, (480, 10))
```

Comment:
(480, 10) isn't scalable with the screen. Think about it as being anchored to one side, then offset it from that side, e.g. `(SCREEN_WIDTH-stringConnect.get_rect().width-20` (sub out a number that looks good for 20). Also it may be `.width` or `.width() `. I can't remember.

This applies to the other text below.